content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -0,0 +1,464 @@
+---
+title: Converting an existing Gatsby project to TinaCMS
+last_edited: '2024-10-25T08:29:23.867Z'
+next: content/docs/cli-overview.mdx
+previous: content/docs/contextual-editing/overview.mdx
+---
+
+> ⚠️ Warning: TinaCMS does not officially support Gatsby. We recommend migrating your Gatsby site to a well supported framework such as [Next.JS](https://tina.io/docs/frameworks/next/overview/) instead.
```

Comment:
We have a warning embed/template (`WarningCallout`) – you could use that instead :)

Reply:
Done - I'm using that component now